presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -316,4 +151,184 @@ public String toString()
+    {
+        return new Builder(
+                DEFAULT_STRIPE_MIN_SIZE,
+                DEFAULT_STRIPE_MAX_SIZE,
+                DEFAULT_STRIPE_MAX_ROW_COUNT,
+                DEFAULT_ROW_GROUP_MAX_ROW_COUNT,
+                DEFAULT_DICTIONARY_MAX_MEMORY,
+                DEFAULT_MAX_STRING_STATISTICS_LIMIT,
+                DEFAULT_MAX_COMPRESSION_BUFFER_SIZE,
+                OptionalInt.empty(),
+                new ByStreamSize(),
+                false);
+    }
+
+    public static class Builder
+    {
+        private DataSize stripeMinSize;
+        private DataSize stripeMaxSize;
+        private int stripeMaxRowCount;
+        private int rowGroupMaxRowCount;
+        private DataSize dictionaryMaxMemory;
+        private DataSize maxStringStatisticsLimit;
+        private DataSize maxCompressionBufferSize;
+        private OptionalInt compressionLevel;
+        private StreamLayout streamLayout;
+        private boolean integerDictionaryEncodingEnabled;
+
+        private Builder(
+                DataSize stripeMinSize,
+                DataSize stripeMaxSize,
+                int stripeMaxRowCount,
+                int rowGroupMaxRowCount,
+                DataSize dictionaryMaxMemory,
+                DataSize maxStringStatisticsLimit,
+                DataSize maxCompressionBufferSize,
+                OptionalInt compressionLevel,
+                StreamLayout streamLayout,
+                boolean integerDictionaryEncodingEnabled)
+        {
+            this.stripeMinSize = stripeMinSize;
+            this.stripeMaxSize = stripeMaxSize;
+            this.stripeMaxRowCount = stripeMaxRowCount;
+            this.rowGroupMaxRowCount = rowGroupMaxRowCount;
+            this.dictionaryMaxMemory = dictionaryMaxMemory;
+            this.maxStringStatisticsLimit = maxStringStatisticsLimit;
+            this.maxCompressionBufferSize = maxCompressionBufferSize;
+            this.compressionLevel = compressionLevel;
+            this.streamLayout = streamLayout;
+            this.integerDictionaryEncodingEnabled = integerDictionaryEncodingEnabled;
+        }
+
+        public Builder withStripeMinSize(DataSize stripeMinSize)
+        {
+            this.stripeMinSize = requireNonNull(stripeMinSize, "stripeMinSize is null");
+            return this;
+        }
+
+        public Builder withStripeMaxSize(DataSize stripeMaxSize)
+        {
+            this.stripeMaxSize = requireNonNull(stripeMaxSize, "stripeMaxSize is null");
+            return this;
+        }
+
+        public Builder withStripeMaxRowCount(int stripeMaxRowCount)
+        {
+            checkArgument(stripeMaxRowCount >= 1, "stripeMaxRowCount must be at least 1");
+            this.stripeMaxRowCount = stripeMaxRowCount;
+            return this;
+        }
+
+        public Builder withRowGroupMaxRowCount(int rowGroupMaxRowCount)
+        {
+            checkArgument(rowGroupMaxRowCount >= 1, "stripeMaxRowCount must be at least 1");
```

Comment:
stripeMaxRowCount in the message -> rowGroupMaxRowCount

Reply:
Nice catch, copy/paste mistake :)